test/util_api.go

```diff
@@ -15,3 +19,29 @@ func CheckRespStatus(c *check.C, resp *http.Response, status int) {
 		c.Assert(resp.StatusCode, check.Equals, status, check.Commentf("Response Body: %v", string(body)))
 	}
 }
+
+// GetMetricValue get the specific metrics from /metrics endpoint,
+// if cannot find this metrics, return not found.
+func GetMetricValue(c *check.C, key string) (float64, bool) {
+	var (
+		val   float64
+		found bool
+	)
+
+	resp, err := request.Get("/metrics")
+	c.Assert(err, check.IsNil)
+	defer resp.Body.Close()
+	data, err := ioutil.ReadAll(resp.Body)
+	c.Assert(err, check.IsNil)
+	lines := strings.Split(string(data), "\n")
+	for _, line := range lines {
+		if strings.Contains(line, key) {
+			val, err = strconv.ParseFloat(strings.Split(line, " ")[1], 64)
+			c.Assert(err, check.IsNil)
+			found = true
+			return val, found
+		}
+	}
+
+	return val, found
```

Comment:
directly use `return val, false` and then we have no need to declare variable `found`. WDYT?

Reply:
Agree. Much better!